CruCentralCoast/Community/Ministry Teams/MinistryTeamsTVC.swift

```diff
@@ -0,0 +1,73 @@
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.navigationController?.navigationBar.prefersLargeTitles = true
+        
+        // Uncomment the following line to preserve selection between presentations
+        // self.clearsSelectionOnViewWillAppear = false
+        
+        // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+        // self.navigationItem.rightBarButtonItem = self.editButtonItem
+        
+        let nib = UINib.init(nibName: "CommunityTableViewCell", bundle: nil)
+        self.tableView.register(nib, forCellReuseIdentifier: "comCell")
+        
+    }
+    
+    // MARK: - Table view data source
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
+        return 1
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        return testDataArray.count
+    }
+    
+    override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 260
+    }
+    
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        guard let vc = UIStoryboard(name: "MinistryTeamDetails", bundle: nil).instantiateViewController(withIdentifier: "MinistryTeamDetailsVC") as? MinistryTeamDetailsVC else {
+            assertionFailure("Probably used the wrong storyboard name or identifier here")
+            return
+        }
+        vc.configure(with: self.testDataArray[indexPath.row])
+        self.navigationController?.present(vc, animated: true, completion: nil)
+    }
+    
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "comCell", for: indexPath) as! CommunityTableViewCell
+        
+        
+        // Configure the cell
+        cell.bigLabel.text = testDataArray[indexPath.row].teamTitle
+        cell.bannerImage.image = testDataArray[indexPath.row].teamImage
+        //only returning first team name
+        cell.smallLabel1.text = testDataArray[indexPath.row].teamLeaders[0]
```

Comment:
why only return the first leader name?

Reply:
yes I need to change that

this was a quick fix just to see if it was working

Reply:
should probably still address this in this PR, should be relatively simple